pyop2/base.py

```diff
@@ -3640,17 +3657,22 @@ def name(self):
         """Kernel name, must match the kernel function name in the code."""
         return self._name
 
-    @property
```

Comment:
Since we are changing attribute values, I thought not being a property would be much more clear